README.md

```diff
@@ -102,25 +89,75 @@ cargo install --path . --force --locked --features rustfmt,cargo-fmt
 This will install `rustfmt` in your `~/.cargo/bin`. Make sure to add `~/.cargo/bin` directory to
 your PATH variable.
 
-
 ## Running
 
+You can run `rustfmt --help` for information about available arguments.
+
 You can run Rustfmt by just typing `rustfmt filename` if you used `cargo
 install`. This runs rustfmt on the given file, if the file includes out of line
 modules, then we reformat those too. So to run on a whole module or crate, you
 just need to run on the root file (usually mod.rs or lib.rs). Rustfmt can also
 read data from stdin. Alternatively, you can use `cargo fmt` to format all
 binary and library targets of your crate.
```

Comment:
This is the section that I'm most interested in updating to address #4336 

There's items in the current text that are strictly not true, like the `if the file includes out of line modules, then we reformat those too` which is quite false with rustfmt 2.0 (so relevant for anyone building from source). Then there's other items like the `if you used cargo install` clause in the first sentence feels a bit dated/out of place (`rustfmt filename` is available regardless of whether you used rustup, installed from crates.io, or build from source).

Here's a high level framing and order of the items that IMO we should call out here, let me know what you think

1. `cargo fmt` is the easiest way to run rustfmt against your entire project and is the general recommendation, worth calling out `cargo fmt --help`. this is true for single-crate projects, but also cargo workspaces (not sure we necessarily need to call that out here)
2. Less commonly used, but rustfmt can also be used directly to format files (`rustfmt foo.rs bar.rs`) or via stdin, but want to highlight the stdin bit more than the current text does
    a. rustfmt 1.x will format out of line mods by default, so `rustfmt rootfile` will basically format the entire project
    b. rustfmt 2.x does not format out of line mods by default, requires `--recursive` flag

Reply:
That sounds great to me, are you okay with the addition of this in a followup PR where I will also add changes to the `running` section?

Reply:
Yeah that's fine, but will want to keep #4336 open til after that

Reply:
sgtm